backend/app/adapter/facebook/identityprovider.go

```diff
@@ -68,6 +88,12 @@ func (g IdentityProvider) RequestAccessToken(authorizationCode string) (accessTo
 	return apiRes.AccessToken, nil
 }
 
+type fbAccessTokenResponse struct {
```

Comment:
Let's move this struct definition into `RequestAccessToken`. It's only being referenced inside that function. Adopting minimal scope methodology prevents other maintainers from being confused by relevant information.